### DISCUSSION TITLE
Feature Request: 개인 폴더 작성

### DISCUSSION THREAD

**Discussion by Dnadit** [young1ll/kassandra#7] [Announcements]
Fix
Add
Update
Feature Request
Chore
...
머리말 지켜서 개인 폴더 생성해주세요

**Comment by young1ll** [comment]
### 커밋메시지 규칙
> 참고: https://velog.io/@jiheon/Git-Commit-message-%EA%B7%9C%EC%B9%99#Type

`feat:` 새로운 기능 추가, 기존의 기능을 요구 사항에 맞추어 수정
`fix:` 기능에 대한 버그 수정
`build:` 빌드 관련 수정
`chore:` 패키지 매니저 수정, 그 외 기타 수정 ex) .gitignore
`ci:` CI 관련 설정 수정
`docs:` 문서(주석) 수정
`style:` 코드 스타일, 포맷팅에 대한 수정
`refactor:` 기능의 변화가 아닌 코드 리팩터링 ex) 변수 이름 변경
`test:` 테스트 코드 추가/수정
`release:` 버전 릴리즈

찾아보니, 이슈에는 머리말 규칙을 반드시 지키지 않아도 될 것 같습니다. 필요한 경우, 사이드바의 `Label` 사용을 고려해주세요.
